Assets/Scenes/MainMenu.unity

```diff
@@ -1350,6 +1350,11 @@ PrefabInstance:
       propertyPath: m_Name
       value: HowToButton
       objectReference: {fileID: 0}
+    - target: {fileID: 4766358716819435425, guid: de788e9dfccb7ab4eaea0f4d483c2e88,
+        type: 3}
+      propertyPath: m_text
+      value: How to
+      objectReference: {fileID: 0}
```

Comment:
Checked out changes to see what you changed, but still don't get it, what was this change? Asking because I edited the button since it leads to tutorial